src/repository/FileVideoRepository.java

```diff
@@ -39,4 +39,20 @@ public List<Video> findAll() {
         }
         return videos;
     }
+
+    @Override
+    public void delete(Video video) {
+        List<Video> videos = findAll();
+        videos.removeIf(v -> v.getTitulo().equals(video.getTitulo()));
+        try (BufferedWriter bw = new BufferedWriter(new FileWriter(file))) {
+            for (Video v : videos) {
+                bw.write(v.toString());
+                bw.newLine();
+            }
+        } catch (IOException e) {
+            // Ignorar erros por enquanto
```

Comment:
Uma sugestão para os catch que estão sem nada dentro seria colocar alguma mensagem amigável para o usuário entender o que esta acontecendo em um caso de erro.
```suggestion
            System.out.println("Erro ao deletar: " + e.getMessage);
```

Reply:
Implementado!